src/apps/Organization/Label.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { textStyle, unselectable, useTheme } from '@aragon/ui'
+
+const Label = ({ text, block, children }) => {
+  const theme = useTheme()
+  return (
+    <label
+      css={`
+        ${textStyle('label2')};
+        ${unselectable()};
+        color: ${theme.surfaceContentSecondary};
+        display: ${block ? 'block' : 'initial'};
+      `}
+    >
+      {text}
+      {children && children}
```

Comment:
If the first value is truthy, the `&&` tells JavaScript to return the second value:

<img width="126" alt="Screen Shot 2019-09-05 at 09 10 07" src="https://user-images.githubusercontent.com/221614/64344697-f99b0c00-cfbc-11e9-823c-811e4932d92b.png">

If the first value is falsey, the `&&` tells JS to return the first value, without even checking the second:

<img width="416" alt="Screen Shot 2019-09-05 at 09 12 14" src="https://user-images.githubusercontent.com/221614/64344875-50084a80-cfbd-11e9-99fb-579d33b2463d.png">

All this to say: I don't think `children && children` behaves any differently from just a single `children`.